extensions/sftp/processors/ListSFTP.cpp

```diff
@@ -535,9 +535,9 @@ void ListSFTP::listByTrackingTimestamps(
       /* Determine the minimum reliable timestamp based on precision */
       auto minimum_reliable_timestamp = now - listing_lag;
       if (remote_system_timestamp_precision == TARGET_SYSTEM_TIMESTAMP_PRECISION_SECONDS) {
-        std::chrono::floor<std::chrono::seconds>(minimum_reliable_timestamp);
+        minimum_reliable_timestamp = std::chrono::floor<std::chrono::seconds>(minimum_reliable_timestamp);
```

Comment:
This seems like an actual bugfix :+1: , should we file a Jira for this? (and maybe even some tests?)

Reply:
create a jira for it: https://issues.apache.org/jira/browse/MINIFICPP-2176